dashboard/mock-server.js

```diff
@@ -80,3 +80,7 @@ app.get("/metrics/actor/back_pressures", (req, res, next) => {
 app.get("/monitor/await_tree/1", (req, res, next) => {
   res.json(require("./mock/await_tree_1.json"))
 })
+
+app.get("/metrics/back_pressures", (req, res, next) => {
```

Comment:
I suppose timely data is required to test the builtin back-pressures. Providing mock data here does not seem to be helpful.

Since #14361, we're now able to use a real RisingWave cluster to develop the dashboard.

Reply:
Sounds good. Didn't know the #14361 feature before. Will try it later.

> I suppose timely data is required to test the builtin back-pressures. Providing mock data here does not seem to be helpful.

Yeah, when I develop, the static mock data is useless. So I manually change the static data to mock the data change. Will try new feature later.